CRISPResso2/CRISPRessoCORE.py

```diff
@@ -6047,6 +6047,45 @@ def count_alternate_alleles(sub_base_vectors, ref_name, ref_sequence, ref_total_
                             crispresso2_info['results']['refs'][ref_name]['plot_10h_roots'].append(os.path.basename(fig_filename_root))
                             crispresso2_info['results']['refs'][ref_name]['plot_10h_captions'].append("Figure 10f: Quilt of Base Edits for " + args.conversion_nuc_from + 'around cut site for ' + sgRNA_legend + ". Nucleotides are indicated by unique colors (A = green; C = red; G = yellow; T = purple). Substitutions are shown in bold font. Red rectangles highlight inserted sequences. Horizontal dashed lines indicate deleted sequences. The vertical dashed line indicates the predicted cleavage site.")
                             crispresso2_info['results']['refs'][ref_name]['plot_10h_datas'].append([('Allele frequency table', os.path.basename(base_edit_allele_filename))])
+
```

Comment:
This whole chunk was a result from resolving a merge conflict... I'm not confident that it is correct (and it is pretty late). @mbowcut2 could you confirm that this is correct?